src/app/app.component.html

```diff
@@ -15,7 +15,7 @@ <h2><a target="_blank" rel="noopener" href="https://angular.io/tutorial">Tour of
   <li>
     <h2><a target="_blank" rel="noopener" href="https://angular.io/cli">CLI Documentation</a></h2>
   </li>
-
+  <br />HELLO <br />
```

Comment:
Why you use BR.

Reply:
Please consider this and please merge this.